docs/CONTRIBUTING.md

```
@@ -1,37 +1,94 @@
+Run database migrations:
+
+```bash
+python manage.py migrate
+```
+
+(Optional) Start Tailwind watcher:
+```bash
+python manage.py tailwind watch
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)